testdata/expected/docker/java-error-tolerance/org.srclib.srclib-java/unresolved-maven-deps/JavaArtifact.graph.json

```diff
@@ -492,96 +492,68 @@
       "End": 403
     },
     {
-      "DefRepo": "github.com/apache/commons-csv",
```

Comment:
It seems like previously this reference correctly pointed to a separate repository (`github.com/apache/commons-csv`), but now it thinks the definition is within this repository? Is this a regression or an intended behavior change?